cmd/postcli/README.md

```diff
@@ -63,6 +70,59 @@ Example
 
 ```
 
+### Initialize subset
+It is possible to initialize only subset of the files. It's useful if one wants to split initialization between many machines.
+#### Example - split initialization between 2 machines
+Let's assume one wants to initialize 100 units and split the process of initialization to machine A and B.
+
+First let's see how many files it would be:
+```bash
+./postcli -numUnits 100 -printNumFiles
+1600
+```
+
+Next, on machine A:
+```bash
+./postcli -numUnits 100 -id <id> -commitmentAtxId <id> -toFile 799 -datadir ./dataA
+```
+We get the first half - 800 binary files
+```bash
+ls -la ./dataA/*.bin | wc -l
+800
+```
+
+Next, on machine B:
+```bash
+./postcli -numUnits 100 -id <id> -commitmentAtxId <id> -fromFile 800 -datadir ./dataB
+```
+We get the first half - 800 binary files
+```bash
+ls -la ./dataB/*.bin | wc -l
+800
+```
+
+Finally we can combine both sets together. A dummy example to get the feeling. Realisticly it would probably be copying over the network
+```bash
+cp ./dataA/* ./data/
+cp ./dataB/* ./data/
+ls -la ./data/*.bin | wc -l
+1600
+```
+
+**An optional step to select best possible VRF nonce**
+Normally, when `postcli`initializates from the start to the end it will automatically pick the best vrf nonce. The best means pointing to **the label with the smallest value**. It might be important in the future if we allow increasing POS data size, but it is not critical at the moment.
```

Comment:
```suggestion
Normally, when `postcli`initializates from the start to the end it will automatically pick the best VRF nonce. The best means pointing to **the label with the smallest value**. This will avoid a longer initialization time when a node increases their PoST size in the future (not supported yet).
```